kodilla-stream/src/main/java/com/kodilla/stream/world/Continent.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.stream.world;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Continent {
+    private final List<Country> countries = new ArrayList<>();
+
+    public List<Country> getCountries() {
+        return countries;
+    }
+
+}
```

Comment:
brakuje w Continent sposubu na dodawanie Countries.  Obecnie to zawsze bedzie pusta lista. Potrzebujesz dodac settet/konstruktor z parametrem/dodac kraje w konstruktorze